sdk/include/opentelemetry/sdk/common/attribute_utils.h

```diff
@@ -42,8 +65,12 @@ struct AttributeConverter
   OwnedAttributeValue operator()(double v) { return OwnedAttributeValue(v); }
   OwnedAttributeValue operator()(nostd::string_view v)
   {
-    return OwnedAttributeValue(std::string(v));
+    return OwnedAttributeValue(std::string(v.data(), v.size()));
```

Comment:
Are `std::string(v)` and `std::string(v.data(), v.size())` just the same?

Reply:
Made the change to follow Max's PR [here](https://github.com/open-telemetry/opentelemetry-cpp/commit/1e7b9d8a950649809d3957a00e230e43b59a1c86#diff-8ac8b11ea3bd653dfa6b11b6f68ae56fcc246a90e4df905268f73ef65a32e85cR52) (cc @maxgolov )

Reply:
Both should have the same effect. The first form should use [this operator](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/api/include/opentelemetry/nostd/string_view.h#L58).

Reply:
Ah okay, I'll revert it to the old `OwnedAttributeValue(std::string(v));` implementation then